test/adapter/json_api/resource_type_config_test.rb

```diff
@@ -36,22 +41,29 @@ def with_jsonapi_resource_type type
           end
 
           def test_config_plural
-            with_adapter :json_api do
-              with_jsonapi_resource_type :plural do
-                hash = ActiveModel::SerializableResource.new(@comment).serializable_hash
-                assert_equal('comments', hash[:data][:type])
-              end
+            with_jsonapi_resource_type :plural do
+              hash = serializable(@comment, adapter: :json_api).serializable_hash
+              assert_equal('comments', hash[:data][:type])
             end
           end
 
           def test_config_singular
-            with_adapter :json_api do
-              with_jsonapi_resource_type :singular do
-                hash = ActiveModel::SerializableResource.new(@comment).serializable_hash
-                assert_equal('comment', hash[:data][:type])
-              end
+            with_jsonapi_resource_type :singular do
+              hash = serializable(@comment, adapter: :json_api).serializable_hash
+              assert_equal('comment', hash[:data][:type])
             end
           end
+
+          def test_explicit_type_value
+            hash = serializable(@author, serializer: ProfileTypeSerializer, adapter: :json_api).serializable_hash
+            assert_equal('profile', hash.fetch(:data).fetch(:type))
```

Comment:
Then why not use it [there](https://github.com/rails-api/active_model_serializers/pull/1213/files#diff-2e5487b4844a235b4d646ae346b76499R53)?